rebar.config

```diff
@@ -5,7 +5,7 @@
             debug_info,
             warn_untyped_record]}.
 
-{deps, [getopt, {lager, "~>3.6.0"}]}.
+{deps, [getopt, {lager, "~>3.7.0"}]}.
```

Comment:
Can you please rebase the branch? It has another unrelated branch merged into it.